### PR TITLE
Feature/1197/add table on about material page

### PR DESCRIPTION
**Pull request by OlehProtsak**
## Description

Create a table for front-end part of "About Material" page with static data.

#### Screenshots

|         Original          |             
| :-----------------------: | 
| ** 
зображення
** 

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/pages/material/material-about.js [hunk 1]
```diff
@@ -1,28 +1,67 @@
...
+      deleteHandler={() => null}
+      editHandler={() => null}
+    />
+  ));
+
   return (
     <div className={common.container}>
-      <Button
-        data-testid='createMaterialButton'
-        id='add-materials'
```
Comment (Skrekliam) [inline src/pages/material/material-about.js:18]: Please use camel case

File: src/pages/material/material-about.js [hunk 2]
```diff
@@ -1,28 +1,67 @@
...
-        variant='contained'
-        color='primary'
-      >
-        {CREATE_MATERIAL_TITLE_BLOCK}
-      </Button>
+      <div className={styles.header}>
+        <Typography
+          variant='h1'
+          className={styles.materialTitle}
+          data-cy='main-header'
```
Comment (Skrekliam) [inline src/pages/material/material-about.js:45]: Here the same

**Review by Skrekliam — CHANGES_REQUESTED** [review]

File: src/pages/material/material-about.js (hunk 2 above)
Comment (OlehProtsak) [reply src/pages/material/material-about.js:45]: fixed

File: src/pages/material/material-about.js (hunk 1 above)
Comment (OlehProtsak) [reply src/pages/material/material-about.js:18]: fixed

**Review by maarekua — APPROVED** [review]

**Review by Skrekliam — APPROVED** [review]

**Pull request merged**